src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -111,11 +111,11 @@ private string[] SuggestStrings(string expression, PowerFxConfig config, string
         // BoolLitNodeSuggestionHandler
         [InlineData("true|", "true")]
         [InlineData("tru|e", "true", "Trunc")]
```

Comment:
>InlineData [](http://example.com/codeflow?start=9&length=10)

just curious - any reason for the changes in this file? look like same content but just reordered.